python/seldon_core/microservice.py

```diff
@@ -243,26 +243,26 @@ def main():
             [tag.strip() for tag in os.environ.get("JAEGER_EXTRA_TAGS", "").split(",")],
         )
     )
-    logger.info("Parse JAEGER_EXTRA_TAGS %s", jaeger_extra_tags)
+    logger.info("Parse JAEGER_EXTRA_TAGS: {}".format(jaeger_extra_tags))
     # set up log level
     log_level_raw = os.environ.get(LOG_LEVEL_ENV, args.log_level.upper())
     log_level_num = getattr(logging, log_level_raw, None)
     if not isinstance(log_level_num, int):
-        raise ValueError("Invalid log level: %s", args.log_level)
+        raise ValueError("Invalid log level: {}".format(args.log_level))
 
     logger.setLevel(log_level_num)
-    logger.debug("Log level set to %s:%s", args.log_level, log_level_num)
+    logger.debug("Log level set to {}:{}".format(args.log_level, log_level_num))
 
     annotations = load_annotations()
-    logger.info("Annotations: %s", annotations)
+    logger.info("Annotations: {}".format(annotations))
```

Comment:
Why are all the logging lines changed to format?

Reply:
I think if we're going to change it to something it would be more preferable to move to fstrings

Reply:
Can move to `fstrings`. The problem is that these didn't work at all and was flooding terminal with lots of errors.

Reply:
Hmm... not in this specific example, but in few cases these way was throwing lots of errors.

Reply:
Okay, found it. It was `logging.info("starting:", " ".join(cmd))` and I just went step too far adjusting all of them.

Reply:
Reverting fix in most places and adding fix only to ones that do not work.